### PR TITLE
Fixing lab build for SSHDebugPS

### PR DESCRIPTION
**Pull request by rajkumar42**

### CODE REVIEW

**Comment by rajkumar42** [comment]
@faxue-msft @caslan @gregg-miskelly

File: src/SSHDebugPS/SSHDebugPS.csproj [hunk 1]
```diff
@@ -120,11 +136,6 @@
       <Name>Microsoft.VisualStudio.Debugger.Interop.UnixPortSupplier.DesignTime</Name>
       <EmbedInteropTypes>True</EmbedInteropTypes>
     </ProjectReference>
-    <ProjectReference Include="..\prebuild\prebuild.csproj">
```
Comment (gregg-miskelly) [inline src/SSHDebugPS/SSHDebugPS.csproj]: Did you mean to remove this?

**Comment by gregg-miskelly** [comment]
Otherwise LGTM

**Comment by faxue-msft** [comment]
LGTM

**Pull request merged**